save-backend/build.gradle.kts

```diff
@@ -3,6 +3,14 @@ import  org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm")
+    id("org.jetbrains.kotlin.plugin.jpa") version Versions.kotlin
+    id("org.jetbrains.kotlin.plugin.allopen") version Versions.kotlin
+}
+
+allOpen {
+    annotation("javax.persistence.Entity")
+    annotation("javax.persistence.Embeddable")
+    annotation("javax.persistence.MappedSuperclass")
```

Comment:
This additional configuration can be extracted to `SpringBootConfiguration` in buildSrc, when you update the branch. `configureSpringBoot` can have an argument `withSpringDataJpa`, when it's `true` it adds compiler plugin and dependencies and stuff.

Reply:
Added to SpringBootConfiguration